test/serializer.js

```diff
@@ -934,6 +949,9 @@ describe('JSON API Serializer', function () {
         type: 'addresses'
       });
 
+      expect(json.included[1]).to.have.property('id')
+        .equal('123');
+
```

Comment:
Same question here

Reply:
Added expectations that id in an array will be a string as well.